src/@types/Commands.ts

```diff
@@ -0,0 +1,13 @@
+export enum Command {
+  bold = 'bold',
+  italic = 'italic',
+  underline = 'underline',
+  strikethrough = 'strikethrough',
+}
```

Comment:
By convention in this project we only allow a single, default export from `@types` files. You could move this into a separate `@types` file though.

I would also suggest renaming to `FormattingCommand` or `CommandStateValue` to avoid getting it mixed up with the concept of a command that is used in the CommandPalette component.

Reply:
Ah, sorry. I didn't notice that convention at first, but I like it. Moved and renamed: 80b6d09600c9355b9329246f38534b9f38ffca88